packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -74,6 +104,15 @@ export default function Sidebar({
     [sidebarExpanded, sidebarRendered, activePage],
   );
 
+  useEffect(() => {
+    if (squadVersion !== SquadVersion.Off) {
+      trackEvent({
+        event_name: 'impression',
+        target_type: 'squads',
```

Comment:
Fixed according to my changes in the taxonomy.
Please also add target_id as the variation of the button

```suggestion
        target_type: 'create squad',
```